runners/console/index.ts

```diff
@@ -254,6 +254,25 @@ export class Console extends Runner {
         return result;
     }
 
+    runDownloadFile(step: Step, command: Command): RunResult {
+        let result = new RunResult();
+        result.returnCode = 0;
+
+        let installDir = this.getVariable(this.workspaceDirectory);
```

Comment:
Variable names should be descriptive. For example, downloadDir would be a better name.